WcaOnRails/app/webpacker/components/CompetitionForm/SeriesComps.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/no-danger */
```

Comment:
If a rule is only being violated once or twice, I'm guessing it would be better practice to only disable individual lines: `eslint-disable-next-line`?

Reply:
Yeah, ill write a todo comment to remind myself to go through all of these once the PR is almost done, as I have just left them disabled for ease of development while I work